src/Nancy.Tests.Functional/Modules/PerRouteAuthModule.cs

```diff
@@ -19,21 +17,14 @@ public PerRouteAuthModule()
 
             Get["/requiresclaims"] = _ =>
             {
-                this.RequiresClaims(new[] { "test", "test2" });
+                this.RequiresClaims(c => c.Type == "test", c => c.Type == "test2");

```

Comment:
Perhaps we could have an overload of `RequireClaims` that takes a `params string[] claim`. The overload would iterate over the params and compare them against the `Type` of each claim. It would give us a nicer syntax. Thoughts?